kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -56,7 +56,8 @@ window.SidebarView = BaseView.extend({
 
     events: {
         "click .sidebar-tab": "toggle_sidebar",
-        "click .fade": "check_external_click"
+        "click .fade": "check_external_click",
+        "click .sidebar-back": "sidebar_back_one_level"
```

Comment:
It was part of `TopicInnerContainer` so I ignored it. Never noticed that it was not being used anywhere.